common/src/main/java/io/netty/util/internal/ThrowableUtil.java

```diff
@@ -54,16 +54,9 @@ public static String stackTraceToString(Throwable cause) {
         }
     }
 
-    public static boolean haveSuppressed() {
-        return PlatformDependent.javaVersion() >= 7;
-    }
-
     @SuppressJava6Requirement(reason = "Throwable addSuppressed is only available for >= 7. Has check for < 7.")
     public static void addSuppressed(Throwable target, Throwable suppressed) {
-        if (!haveSuppressed()) {
-            return;
-        }
-        target.addSuppressed(suppressed);
```

Comment:
@kezhenxu94 I think this change is not correct and it should always call `target.addSuppressed(...)`. Please fix